device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/SasTokenRenewalHandler.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright (c) Microsoft. All rights reserved.
+ *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ */
+
+package com.microsoft.azure.sdk.iot.device.transport.amqps;
+
+import com.microsoft.azure.sdk.iot.device.CustomLogger;
+import com.microsoft.azure.sdk.iot.device.DeviceClient;
+import com.microsoft.azure.sdk.iot.device.DeviceClientConfig;
+import com.microsoft.azure.sdk.iot.device.exceptions.TransportException;
+import org.apache.qpid.proton.engine.BaseHandler;
+import org.apache.qpid.proton.engine.Event;
+
+public class SasTokenRenewalHandler extends BaseHandler
```

Comment:
Prefix this class with Amqp as this is amqp specific handler and cannot be reuse with other protocols